### ISSUE TITLE
Base64 Compile Error

### ISSUE THREAD

**Issue by michaelkleinhenz** [Qrome/printer-monitor#28]
When I compile latest master against latest libraries, I get:

```
sketch/OctoPrintClient.cpp: In member function 'void OctoPrintClient::updateOctoPrintClient(String, String, int, String, String)':
OctoPrintClient.cpp:38: error: no matching function for call to 'base64::encode(String&, bool)'
     encodedAuth = b64.encode(userpass, true);
                                            ^
sketch/OctoPrintClient.cpp:38:44: note: candidates are:
In file included from sketch/OctoPrintClient.h:27:0,
                 from sketch/OctoPrintClient.cpp:24:
/home/kleinhenz/.arduino15/packages/esp8266/hardware/esp8266/2.3.0/cores/esp8266/base64.h:30:23: note: static String base64::encode(uint8_t*, size_t)
         static String encode(uint8_t * data, size_t length);
                       ^
/home/kleinhenz/.arduino15/packages/esp8266/hardware/esp8266/2.3.0/cores/esp8266/base64.h:30:23: note:   no known conversion for argument 1 from 'String' to 'uint8_t* {aka unsigned char*}'
/home/kleinhenz/.arduino15/packages/esp8266/hardware/esp8266/2.3.0/cores/esp8266/base64.h:31:23: note: static String base64::encode(String)
         static String encode(String text);
                       ^
/home/kleinhenz/.arduino15/packages/esp8266/hardware/esp8266/2.3.0/cores/esp8266/base64.h:31:23: note:   candidate expects 1 argument, 2 provided
exit status 1
[Error] Exit with code=1
```

Any ideas?

**Comment by wabbitguy** [comment]
I had the same thing with the older 2.3.0 libraries. Had to use the Boards Manager in the Arduino IDE to update to 2.4.1 to get rid of the base64 error.

**Comment by Qrome** [comment]
Correct.   base64 is part of the ESP8266 core and is included in 2.4.1 -- update your library files for ESP8266.
